httpcore/_async/connection_pool.py

```diff
@@ -294,3 +294,12 @@ async def aclose(self) -> None:
         # Close all connections
         for connection in connections:
             await connection.aclose()
+
+    def get_connection_info(self) -> Dict[str, List[str]]:
+        """Returns dict with ."""
```

Comment:
Looks like this one didn't get finished, a suggestion based on what I can see here…

```suggestion
        """Returns a dict containing information about the connection of each origin URL."""
```

Reply:
Woops 👍